assets/src/js/frontend/paypal-commerce/AdvancedCardFields.js

```diff
@@ -21,7 +21,12 @@ class AdvancedCardFields extends PaymentMethod {
 	 */
 	async renderPaymentMethodOption() {
 		if ( ! this.canRenderFields() ) {
-			Array.from( this.form.getElementsByClassName( 'give-paypal-commerce-cc-field-wrap' ) ).forEach( el => el.remove() );
+			Array.from( this.form.getElementsByClassName( 'give-paypal-commerce-cc-field-wrap' ) ).forEach(
+				el => {
+					el.previousElementSibling.remove();
+					el.remove();
+				}
+			);
```

Comment:
What is this doing?

Reply:
fixed: https://github.com/impress-org/givewp/compare/84a9166d253a...8397a831e245